pheps/phep-0001.md

```diff
@@ -0,0 +1,338 @@
+
+* The PHEP editors review your PR for structure, formatting, and other errors. Approval criteria are:
+    * It is sound and complete. The ideas must make technical sense. The editors do not consider whether they seem likely to be accepted.
+    * It is of reasonable scope, not too broad or unfocused.
+    * The title accurately describes the content.
+    * The PHEP's language (spelling, grammar, sentence structure, etc.) should be correct and conformant. Markdown (MD) markup and other formatting should be clear and reasonable.
+
+    Editors are generally quite lenient about this initial review, expecting that problems will be corrected by the reviewing process. Correctness is the responsibility of authors and reviewers, not the editors.
+
+    If the PHEP isn't ready for approval, an editor will send it back to the author with specific instructions.
+
+* Once approved, they will assign your PHEP a number. All PHEPs are sequentially-numbered at time of assignment. "Special" numbers (X, 0, etc.) are not used. Upon approval, the author must mark the PHEP PR ready for review.
+
+The PHEP editors will not unreasonably deny approval of a PHEP. Reasons for denying PHEP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the PyHC philosophy.
+
+Standards Track PHEPs may consist of two parts, a design document and a reference implementation. It is generally recommended that at least a prototype implementation in an existing or new PyHC package be co-developed with the PHEP, as ideas that sound good in principle sometimes turn out to be impractical when implemented.
+
+## Discussing a PHEP
+<a name="discussing-a-phep"></a>
+Once the draft PHEP is submitted in an open PR and a PHEP number has been assigned, the PR discussion thread provides the central place to discuss and review its contents. The PHEP must be updated so that the `Discussions-To` header links to the PR. The discussion must also be announced by the PHEP authors via reasonable means; this includes (but is not necessarily limited to) the PyHC email list and the PyHC Matrix/Slack.
+
+Informal conversation may elsewhere, but the discussion of record is the public thread on the pull request. Substantive input into the PHEP or questions about it belong in the PR comments and side conversations should result in comments on the PR. This ensures everyone can follow and contribute, avoids fragmenting the discussion, and makes sure input is fully considered as part of the PHEP review process. Feedback on the PR thread is a critical part of what the community will consider when reviewing the PHEP. Individual comments or the GitHub "review" process may both be used. Commenters may use the GitHub "approve" feedback to indicate they believe the PHEP is ready for a vote; however, this does not change the PHEP status to "Accepted".
```

Comment:
Caught this typo ("informal conversation may _happen_ elsewhere") and will be fixed next push.

Reply:
@rebeccaringuette expressed support for the "approve" feedback on the PR. It might make sense to move this information down to the review & resolution section? Or maybe just emphasize there that the community feedback up in the discussion area is the place where we really know if there is approval, and then the review / resolution is a formality. I'll see if I can't punch this up.